src/import-data/ImportDataDestination.test.ts

```diff
@@ -44,74 +44,51 @@ jest.mock(
 );
 
 describe('ImportDataDestination', () => {
-  it('should explain protected data restricts eligible workspaces', async () => {
+  it.each([true, false])('should explain protected data restricts eligible workspaces', async (isProtectedData) => {
```

Comment:
@pshapiro4broad re: [your comment on #4227](https://github.com/DataBiosphere/terra-ui/pull/4227#discussion_r1327622456), this reduces duplicated test setup code.